src/rgw/rgw_rest_s3.cc

```diff
@@ -3030,7 +3197,10 @@ RGWOp *RGWHandler_REST_Obj_S3::op_put()
 {
   if (is_acl_op()) {
     return new RGWPutACLs_ObjStore_S3;
+  } else if (is_tagging_op()) {
```

Comment:
I know this is the pattern in the surrounding code, but if we're returning, what is "else" accomplishing?